python/lsst/daf/butler/registry/tests/_registry.py

```diff
@@ -2722,8 +2722,8 @@ def testSkipCalibs(self):
     def testIngestTimeQuery(self):
         butler = self.make_butler()
         registry = butler._registry
-        self.load_data(butler, "base.yaml", "datasets.yaml")
         dt0 = datetime.datetime.now(datetime.UTC)
+        self.load_data(butler, "base.yaml", "datasets.yaml")
```

Comment:
I could split the load_data up into two calls but I'm not sure it makes a difference for this test.

Reply:
I think it's OK for this test to have `dt0` before loading everuthing.